GenderPayGap.WebUI/wwwroot/styles/components/_number-icons.scss

```diff
@@ -0,0 +1,37 @@
+﻿.action-step {
+  position: relative;
+  padding-bottom: 10px;
+}
+
+.action-step .icon {
+  position: absolute;
+}
+
+.action-step h3,
+.action-step p,
+.action-step ul {
+  padding-left: 30px;
+}

```

Comment:
A couple of thoughts:
* Since we're using Sass, we can nest this to make it really clear that the `h3`/`p`/`ul` are within the `.action-step`, like this:
   ```scss
    .action-step {
        position: relative;
        padding-bottom: 10px;

        h3, p, ul {
            padding-left: 30px;
        }
    }
    ```
    In fact, it would be great if **all the CSS** in this file was within `.action-step` so that we don't accidentally re-use it somewhere else (e.g. it would be easy to use the class `icon` somewhere without remembering that it's defined here)

* But, even better, instead of having to list all the elements (`h3`, `p`, `ul`) that we happen to have inside the `.action-step`, it would be nice it we could put anything inside the `.action-step` - e.g.
    ```scss
    .action-step {
        position: relative;
        padding-bottom: 10px;
        padding-left: -30px; // new

        // can remove this section
        //h3, p, ul {
        //   padding-left: 30px;
        //}

        .icon {
            position: absolute;
            margin-left: -30px; // new
        }
    }
    ```